### PR TITLE
[Llama-Vision] Trim logits (#162)

### PR DESCRIPTION
**Pull request by ANSHUMAN87**

### CODE REVIEW

File: optimum/habana/transformers/models/mllama/modeling_mllama.py [hunk 1]
```diff
@@ -908,6 +908,12 @@ def forward(
         )
 
         hidden_states = outputs[0]
+        _, seq_len, _ = hidden_states.shape
+        if seq_len > 1 and not self.training:
```
Comment (yafshar) [inline optimum/habana/transformers/models/mllama/modeling_mllama.py:912]: @ANSHUMAN87 Does your change also require adding `trim_logits` to the forward interface? and do below modification? 
`if seq_len > 1 and trim_logits and not self.training:`

Comment (regisss) [reply optimum/habana/transformers/models/mllama/modeling_mllama.py:912]: I agree with @yafshar, I'm curious about this too :eyes:
